guide-us-english.tex

```diff
@@ -325,20 +325,23 @@ \section{Product Owner Cycle - Coordinating the ``What''}
 
 The PO organization (the Product Owners, the CPO's, and the Executive MetaScrum) work as a whole to satisfy the components of the Product Owner Cycle. The components unique to the Product Owner Cycle are:
 \begin{itemize}
-	\item \textbf{Strategic Vision,}
-	\item \textbf{Backlog Prioritization,}
-	\item \textbf{Backlog Decomposition and Refinement, and}
-	\item \textbf{Release Planning.}
+	\item \textbf{Strategic Vision},
+	\item \textbf{Backlog Prioritization},
+	\item \textbf{Backlog Decomposition and Refinement}, and
+	\item \textbf{Release Planning}.
 \end{itemize}
 
 \subsection{Strategic Vision}
 
+A compelling vision attracts both customers and great employees.
+Therefore, in the PO cycle, a Strategic Vision needs to be formulated and communicated both internally and externally.
+
```

Comment:
I agree there needs to be an intro, but this... isn't it for me. Needs some work.